composer.json

```diff
@@ -34,7 +34,7 @@
     "require": {
         "php": "^7.1 || ^8",
         "ext-pdo": "*",
-        "doctrine/cache": "^1.0",
+        "doctrine/cache": "^1.0|^2.0",
```

Comment:
For code this is fine, but for tests we'll have to require `^1.11|^2.0` to avoid `--prefer-lowest` builds failing because 1.0 doesn't contain the PSR-6 adapters.

Reply:
If the adapter is not there, the tests fall back to `ArrayCache`, so we're good.

Reply:
🤦‍♂️ Of course, yes.